medical-portal/src/UI/src/app/case-submissions/case-submissions.component.html

```diff
@@ -67,7 +67,7 @@
                 <div>
                   <label>Submission Date</label>
                   <div>
-                    <b>{{ getFormattedDate(document.dueDate) }}</b>
+                    <b>{{ getFormattedDate(document.faxReceivedDate) }}</b>
```

Comment:
I haven't read the requirements, just want to double check that the "Fax Received Date" on Dynamics is what they want to use for the "Submission Date" on the website?

Reply:
Yes Jebb, This is not the compliance Date on the document but the UploadedDate(dfp_faxreceiveddate) on the document after testing. The requirement was in 156 Story comments

Reply:
Do we also need "DueDate"? As is, this will be very confusing for the next developer, which may happen soon. ComplianceDate is mapped to DueDate and FaxReceivedDate is mapped to FaxReceivedDate but used like DueDate.

Reply:
Yes we need both the dates Due Date to show on the Case submissions Tab. The FaxReceivedDate which is submissiondate Field on the driver documents tab.

Reply:
The field should be called "UploadedDate" then outside of cms-adapter. Only cms-adapter should have the property name "FaxReceivedDate" on the internal models. At the very least, document these fields to help avoid confusion

Reply:
Agree Jebb I can change that to uploaded Date but at this point I am using the API names and also be consistent with other document mappings. It would be more confusing to use Uploaded date for one mapping and FaxReceivedDate  for other mapping.